src/main/kotlin/com/jakewharton/trakt/ratingsremover/main.kt

```diff
@@ -66,6 +67,9 @@ private class MainCommand : CliktCommand(
 				chain.proceed(requestBuilder.build())
 			}
 			.addNetworkInterceptor(HttpLoggingInterceptor(debug::log).setLevel(BASIC))
+			.connectTimeout(60, TimeUnit.SECONDS)
+			.writeTimeout(120, TimeUnit.SECONDS)
+			.readTimeout(60, TimeUnit.SECONDS)
```

Comment:
If I had to guess, the `readTimeout` change is the only one that is needed.

"connect" is how long it takes to establish a connection to the server. This should always be fast. "write" is how long it takes to send our request. This also should always be relatively fast. "read" is how long we wait to read the response from the request. If you sent a few thousand actions for it to perform, it's likely not going to reply for quite some time and this triggers the default 10 second timeout.

Does that sound right?

```suggestion
			.readTimeout(60, TimeUnit.SECONDS)
```